Sources/third_party/Experimental/Complex.swift

```diff
@@ -1,5 +1,55 @@
-// A major part of the Complex number implementation comes from
-// [@xwu](https://github.com/xwu)'s project, [NumericAnnex](https://github.com/xwu/NumericAnnex).
+//
+//  Complex.swift
+//  NumericAnnex
+//
+//  Created by Xiaodi Wu on 3/25/17.
+//
```

Comment:
Maybe replace lines 1-5 with

```swift
// Copyright 2017-2019 Xiaodi Wu and The TensorFlow Authors. All Rights Reserved.
//
// Licensed under the MIT License (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://opensource.org/licenses/MIT
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
So we are including TensorFlow Authors instead of Google LLC? I was even thinking maybe `Copyright 2017-2019 Xiaodi Wu, The TensorFlow Authors, and Google LLC`

Reply:
And would I then remove lines 14-16:
```swift
//  Code in libc++ is dual-licensed under the MIT and UIUC/NCSA licenses.
//  Copyright © 2009-2017 contributors to the LLVM/libc++ project.
/// A type to represent a complex value in Cartesian form.
```

Reply:
Since all official TensorFlow source files use `The TensorFlow Authors` and because we are a TensorFlow repo, I feel we should use `The TensorFlow Authors` instead.